spec/axe.spec.js

```diff
@@ -20,6 +20,9 @@ const AXE_OPTIONS = JSON.stringify({
     // ignore that rule.
     'bypass': { enabled: false },
   },
+  exclude: [
+    [ 'iframe[src^="data:"]' ],
```

Comment:
I'm not sure why, but aXe takes a lot longer to run if it needs to check iframes, so I'm explicitly telling it not to dive into iframes with a `data:` URI source here.